daemon/cluster/executor/container/executor.go

```diff
@@ -154,9 +159,35 @@ func (e *executor) Controller(t *api.Task) (exec.Controller, error) {
 		return newNetworkAttacherController(e.backend, t, e.secrets)
 	}
 
-	ctlr, err := newController(e.backend, t, secrets.Restrict(e.secrets, t))
-	if err != nil {
-		return nil, err
+	var ctlr exec.Controller
```

Comment:
Nevermind, apparently a node can only have one executor according to `daemon/cluster/noderunner.go` (search for `NewExecutor`)